packages/flutter_tools/templates/create/lib/main.dart.tmpl

```diff
@@ -30,18 +29,24 @@ class MyApp extends StatelessWidget {
         // reset back to zero -- the application is not restarted.
         primarySwatch: Colors.blue,
       ),
-      home: new MyHomePage(),
+      home: new MyHomePage(title: 'Flutter Demo'),
     );
   }
 }
 
 class MyHomePage extends StatefulWidget {
-  MyHomePage({Key key}) : super(key: key);
+  MyHomePage({ Key key, this.title }) : super(key: key);
```

Comment:
We've been using dartfmt style for the template so it doesn't change when the developer autoformats it.